webpack/assets/javascripts/react_app/redux/reducers/hosts/storage/vmware.js

```diff
@@ -91,12 +95,13 @@ export default (state = initialState, { type, payload }) => {
         controllers: payload.controllers,
         paramsScope: payload.config.paramsScope,
         datastores: [],
-        datastoresLoading: true,
+        datastoresLoading: false,
```

Comment:
Why change this?

Reply:
Because now it is a default state. The loading of datastores is not triggerd untill cluster is selected.